test/e2e/resize_nodes.go

```diff
@@ -330,7 +330,14 @@ func performTemporaryNetworkFailure(c *client.Client, ns, rcName string, replica
 	// and cause it to fail if DNS is absent or broken.
 	// Use the IP address instead.
 
-	iptablesRule := fmt.Sprintf("OUTPUT --destination %s --jump DROP", testContext.CloudConfig.MasterName)
+	destination := testContext.CloudConfig.MasterName
+	if providerIs("aws") {
+		// TODO: Use for all clouds?
+		// TODO: Avoid hard-coding this
+		destination = "127.20.0.9"
```

Comment:
Ah... https://github.com/justinsb/kubernetes/commit/77f8c1fc3d528d5d65b2643112d5c6385f2301df  I am going to fix that - #10745 - and then come back and fix this.  Marking this one as WIP.